application/templates/stage-count.html

```diff
@@ -1,10 +1,5 @@
 {% extends 'layouts/base.html' %}
 
-
-{% block beforeContent %}
-<a href="{{ url_for('main.index') }}" class="govuk-back-link">Back</a>
-{% endblock beforeContent %}
```

Comment:
Don't we want the back button?

Reply:
I believe you can do:

```
{% block beforeContent %}
  {{ super() }}
  <a href="{{ url_for('main.index') }}" class="govuk-back-link">Back</a>
{% endblock beforeContent %}
```

`{{ super() }}` pulls in the content from the parent template’s beforeContent block.